hermes_eea/tests/test_calibration.py

```diff
@@ -54,7 +54,7 @@ def test_process_file(small_level0_file):
             # Process the File
             output_files = calib.process_file(temp_test_file_path)
 
-            assert os.path.getsize(output_files[0]) > 275000
+            assert os.path.getsize(output_files[0]) > 274000
```

Comment:
The files size using the in-built spacepy cdf library was `274038` causing this test to fail, @rstrub could you verify the processed file is still correct and matches what you expect in your tests? ty!

Reply:
after lowering the test to `274000` it passed

Reply:
Why would this change? Is this a new version of the cdf library that is leading to this? If the file size might change based on the library we use to write the file then it may not make sense to have a test that is so sensitive. Perhaps just test that the file is above a very small value so that we can be sure that it has contents.

Reply:
Yeah it looks like it uses 3.9.1 version of the cdf library which is a minor update. I'm not sure if there's a way we can confirm the file is correct other than it's file size since that might be a better approach @rstrub, either verifying a few values or headers?